doc/train_parameters.md

```diff
@@ -0,0 +1,155 @@
+# Brief design for unifying format of training parameters in SQLFlow
+
+SQLFlow propose an extension to the SQL SELECT statement, which proposes the syntax for training as 
```

Comment:
SQLFlow extends the syntax of the SELECT statement of SQL to support training a model:

Reply:
Thanks, fixed.